### PR TITLE
[Bobrandy13] Created Contributions Page

### PR DESCRIPTION
**Pull request by bobrandy13**
Hello, 

I made the contributions page.

Please let me know what to change and I am happy to change it. Will maybe begin working on the dynamic data fetching? 

Thanks

### CODE REVIEW

**Comment by MathPow** [comment]
Need help here ->
I wanted do the rebase to help and check the changes locally, but I don't know how can I checkout a branch that is from another forked branch. Do you guys know how?

**Comment by bobrandy13** [comment]
Hm, I not sure but I had a look online. 
https://stackoverflow.com/questions/9153598/how-do-i-fetch-a-branch-on-someone-elses-fork-on-github
Does this work?

**Comment by ShaanCoding** [comment]
I've done a pretty thorough search and I can't find another other OSS projects which have similar layouts. Not sure on a good layout to show user info and also contain a list of contributions they've done. But I personally think a card may not be enough data / be too minimal for the page.

Not sure on the actionables here yet, but if you're able to find other examples of OSS projects with websites showing their contributors for inspiration or an excalidraw wireframe of some proposed UI/UX ideas that be great.

Until then I'll be keeping this one in the backlog as I'm not sure a good way to handle this ticket.

Closest I found was this standard https://github.com/all-contributors/all-contributors which contributes all contributors and categories them into:

image
Available here: https://allcontributors.org/docs/en/emoji-key

We could do like tabs for the main ones i.e. Dev, Documentation, Templates, Bug Reports and then show cards / a row with their name, contribution, and what they did / link PRS / a github card like this:

image

And then some comments like

MathPow:
- #44 
- #58 
- #67 
- #81 

For example

**Comment by bobrandy13** [comment]
oops mb team

File: frontend/app/contributions/contributions.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+export default function contributions() {
+  // TODO: returns a list of contributions dynammically fetched from GITHUB
```
Comment (bobrandy13) [inline frontend/app/contributions/contributions.ts:2]: oops this file shouldnt be here.

**Comment by ShaanCoding** [comment]
Had a review & overall LGTM!

I will have to do some minor tweaks tomorrow in regards to type safety & moving types to backend interface w/ Zod & regenerate frontend types but the UI/UX looked very stable. I've just done some minor changes to the CSS classes to be a bit more consistent with other pages so the movement is a bit more seamless (this is also a WIP and requires a tiny bit more rework till it's good)

**Comment by bobrandy13** [comment]
i feel bad the amount of code you have to refactor :sob: so much to learn

**Pull request merged**